ingestion/src/metadata/utils/sql_queries.py

```diff
@@ -242,3 +242,23 @@
          order by dbg.name
         """
 )
+
+VERTICA_GET_COLUMNS = """
+        SELECT column_name, data_type, column_default, is_nullable, comment
+        FROM v_catalog.columns col left join v_catalog.comments com on com.object_name=lower(col.table_name) and com.object_type='COLUMN' and col.column_name=com.child_object  
+        WHERE lower(table_name) = '%(table)s'
+        AND %(schema_condition)s
+        UNION ALL
+        SELECT column_name, data_type, '' as column_default, true as is_nullable, ''  as comment
+        FROM v_catalog.view_columns
+        WHERE lower(table_name) = '%(table)s'
+        AND %(schema_condition)s
+    """
+
+VERTICA_GET_PRIMARY_KEYS = """
```

Comment:
@ulixius9 can we get other constraints as well like unique key for example

Reply:
@harshach I've updated the query, now using table id instead of table name